src/muler/igrins.py

```diff
@@ -140,24 +154,31 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
     @staticmethod
-    def read(file, precache_hdus=True):
+    def read(file, wavefile=None, precache_hdus=True):
```

Comment:
Same as above--- move the `wavefile=None` to the last kwarg, to de-emphasize this pattern for most stellar astronomers.

Reply:
Ditto. Moved wavefile to the end.